lib/fakerbot/version.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
+
 module FakerBot
-  VERSION = '0.4.1'.freeze
+  VERSION = '0.4.1'
```

Comment:
Let's bump the version here to `0.4.2` 🙂

Reply:
Done